src/repos/ArticleRepo.ts

```diff
@@ -88,6 +89,27 @@ const refreshFeed = async (): Promise<Article[]> => {
   return articles;
 };
 
+/**
+ * Creates a new article object and inserts it into database.
+ */
+const createArticle = async (
+  title: string,
+  articleURL: string,
+  pubDate: string,
+  pub: string,
+  content: string,
+): Promise<Article> => {
+  const articleObject = Object.assign(new Article(), {
+    articleURL,
+    date: new Date(pubDate),
+    imageURL: content,
+    publicationSlug: pub,
+    title,
+  });
+  const articleDoc = await ArticleModel.insertMany(articleObject, { ordered: false });
```

Comment:
I believe this will crash if the articleURLs provided are the same. Also I think this is used for testing right? If so, keeping the test article in the DB may be a bit problematic.